recipe/meta.yaml

```
@@ -10,7 +10,7 @@ source:
   sha256: 758dbdaa20add2db4902df0b1b7c936564b7376c02a0acd1f2a331bd334b38c7
 
 build:
-  number: 1
+  number: 0
```

```suggestion
  number: 1
```
Won't this make conda-forge's package install at the same preference level as our own?

---

Done